upgrade_tests/test_scenarios/test_subscription.py

```diff
@@ -0,0 +1,106 @@
+"""Test for subscription manifest related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from automation_tools.satellite6 import hammer
+from fabric.api import env, execute
+from nailgun import entities
+from unittest2.case import TestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+from upgrade_tests.helpers.scenarios import get_satellite_host, upload_manifest
+import os
+
+
+class Scenario_manifest_refresh(TestCase):
+    """The test class contains pre-upgrade and post-upgrade scenarios to test
+    manifest refresh before and after upgrade
+
+    Test Steps:
+
+    1. Before Satellite upgrade, upload a manifest.
+    2. Refresh the manifest.
+    3. Upgrade satellite.
+    4. Refresh the manifest.
+    5. Delete the manifest.
+
+    """
+    manifest_url = os.environ.get('MANIFEST_URL')
+    org_name = 'preupgrade_subscription_org'
+
+    @pre_upgrade
+    def test_pre_manifest_scenario_refresh(self):
+        """Pre-upgrade scenario that upload and refresh manifest in satellite
+         which will be refreshed in post upgrade scenario.
+
+
+        :id: preupgrade-29b246aa-2c7f-49f4-870a-7a0075e184b1
+
+        :steps:
+            1. Before Satellite upgrade, upload and refresh manifest.
+
+        :expectedresults: Manifest should upload and refresh successfully.
+         """
+        env.host_string = self.sat_host
+        org = entities.Organization()
+        org.name = self.org_name
+        org = org.create()
+        execute(upload_manifest, self.manifest_url, org.name,
```

Comment:
Not required to execute on sat_host as we already intiated the host with env.host_string
So,
```
upload_manifest(self.manifest_url, org.name)
```